cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -315,7 +327,7 @@ handleSimpleNode p trace nodeTracers npm onKernel = do
          traceWith tracer $
            "System started at " <> show (nodeStartTime (Proxy @blk) cfg)
 
-         traceWith tracer $ unlines
+         traceWith tracer $ Prelude.unlines
```

Comment:
Would it not be better to use `Text.unlines` which specifles exactly what data type the we are operating on.

Since `Text` and `ByteString` export many of the same function names, it is best to always uses those functions qualified, ie as `Text.unlines` or `BS.unlines` or `LBS.unlines`.

Reply:
yes, that 'unlines' was causing warnings and was 'bit awkward. Changing the type of the tracer to 'Text' and using Text.unlines is a very good solution. Thanks!